typescript/src/lib/moderation/structures/ModerationMessageEvent.ts

```diff
@@ -60,7 +60,13 @@ export abstract class ModerationMessageEvent<T = unknown> extends Event {
 		}
 	}
 
-	protected processSoftPunishment(message: GuildMessage, language: TFunction, bitField: SelfModeratorBitField, preProcessed: T) {
+	protected processSoftPunishment(
+		message: GuildMessage,
+		channelId: string | Nullish,
```

Comment:
```suggestion
		logChannelId: string | Nullish,
```

And update references